src/jit/codegenxarch.cpp

```diff
@@ -1855,6 +1855,13 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
                         type0 = TYP_REF;
                         firstIntUsed = true;
                         break;
+                    case SystemVClassificationTypeIntegerByReference:
```

Comment:
Getting of Jit type and size uses utility routine now. Setting the right for each eightbyte has a different code - one sets the firstXXXXXUsed flag, the other is reading the flag.